packages/errors/src/context.ts

```diff
@@ -191,469 +195,471 @@ type Bytes = number;
+                errorEvent: Event;
+            };
+            [SOLANA_ERROR__RPC__API_PLAN_MISSING_FOR_RPC_METHOD]: {
+                method: string;
+                params: readonly unknown[];
+            };
+            [SOLANA_ERROR__RPC__INTEGER_OVERFLOW]: {
+                argumentLabel: string;
+                keyPath: readonly (number | string | symbol)[];
+                methodName: string;
+                optionalPathLabel: string;
+                path?: string;
+                value: bigint;
+            };
+            [SOLANA_ERROR__RPC__TRANSPORT_HTTP_ERROR]: {
+                headers: Headers;
+                message: string;
+                statusCode: number;
+            };
+            [SOLANA_ERROR__RPC__TRANSPORT_HTTP_HEADER_FORBIDDEN]: {
+                headers: readonly string[];
+            };
+            [SOLANA_ERROR__SIGNER__ADDRESS_CANNOT_HAVE_MULTIPLE_SIGNERS]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SIGNER__EXPECTED_KEY_PAIR_SIGNER]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SIGNER__EXPECTED_MESSAGE_MODIFYING_SIGNER]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SIGNER__EXPECTED_MESSAGE_PARTIAL_SIGNER]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SIGNER__EXPECTED_MESSAGE_SIGNER]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SIGNER__EXPECTED_TRANSACTION_MODIFYING_SIGNER]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SIGNER__EXPECTED_TRANSACTION_PARTIAL_SIGNER]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SIGNER__EXPECTED_TRANSACTION_SENDING_SIGNER]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SIGNER__EXPECTED_TRANSACTION_SIGNER]: {
+                address: string;
+            };
+            [SOLANA_ERROR__SUBTLE_CRYPTO__CANNOT_EXPORT_NON_EXTRACTABLE_KEY]: {
+                key: CryptoKey;
+            };
+            [SOLANA_ERROR__TIMESTAMP_OUT_OF_RANGE]: {
+                value: bigint;
+            };
+            [SOLANA_ERROR__TRANSACTION_ERROR__DUPLICATE_INSTRUCTION]: {
+                index: number;
+            };
+            [SOLANA_ERROR__TRANSACTION_ERROR__INSUFFICIENT_FUNDS_FOR_RENT]: {
+                accountIndex: number;
+            };
+            [SOLANA_ERROR__TRANSACTION_ERROR__PROGRAM_EXECUTION_TEMPORARILY_RESTRICTED]: {
+                accountIndex: number;
+            };
+            [SOLANA_ERROR__TRANSACTION_ERROR__UNKNOWN]: {
+                errorName: string;
+                transactionErrorContext?: unknown;
+            };
+            [SOLANA_ERROR__TRANSACTION__ADDRESSES_CANNOT_SIGN_TRANSACTION]: {
+                expectedAddresses: readonly string[];
+                unexpectedAddresses: readonly string[];
```

Comment:
Since these context objects are now automatically readonly-fied, is the convention that we still prefix arrays and object with the `readonly` keyword? I'm happy either way but I think we should be consistent with this.